### PR TITLE
Fpslagsreducerfor mixin render global

### PR DESCRIPTION
**Pull request by quentin452**

### CODE REVIEW

File: src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinRenderGlobal.java [hunk 1]
```diff
@@ -57,39 +59,30 @@ public class MixinRenderGlobal {
                     target = "Lnet/minecraft/client/renderer/tileentity/TileEntityRendererDispatcher;renderTileEntity(Lnet/minecraft/tileentity/TileEntity;F)V"))
     public void hodgepodge$postTESR(TileEntityRendererDispatcher instance, TileEntity j, float k) {
         ManagedEnum<RenderDebugMode> renderDebugMode = HodgepodgeClient.renderDebugMode;
-        if (!renderDebugMode.is(RenderDebugMode.OFF))
-            // this should be enough
-            GL11.glPushAttrib(
```
Comment (eigenraven) [inline src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinRenderGlobal.java:62]: Don't remove the pushattrib/popattrib pair

**Comment by Dream-Master** [comment]
@quentin452

**Comment by quentin452** [comment]
don't have time for now

**Comment by quentin452** [comment]
new pull : https://github.com/GTNewHorizons/Hodgepodge/pull/236

**Pull request closed without merging**